README.md

```diff
@@ -120,7 +120,7 @@ Once you complete the installation, use any of these credentials to login in to
 
 **username**: `admin`, `moderator`, or `user` 
 
-**password**: `password`
+**password**: `password` , `password123` 
```

Comment:
It's not required as only **password** can be used to sign-in. Please refer `db/seeds.rb`.